sdk/core/azure-core-management/src/main/java/com/azure/core/management/SystemData.java

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.management;
+
+import java.time.OffsetDateTime;
+
+/** Metadata pertaining to creation and last modification of the resource. */
+public final class SystemData {
```

Comment:
Is this type missing a constructor? How are the property values set?

Reply:
The class is initialized by Jackson (via reflection), hence no constructor (or just default constructor). All properties is read-only, hence no setter.